users/api/serializers.py

```diff
@@ -56,4 +79,44 @@ def create(self, data):
         user.save()
         email_service = EmailService()
         email_service.send_registration_email(user)
-        return user
\ No newline at end of file
+        return user
+
+
+class UserProductReactionSerializer(serializers.Serializer):
+    product_id = serializers.IntegerField()
+
+
+class UserProductReactionSerializer(serializers.ModelSerializer):
```

Comment:
- Get user from request with serializer fields
- read to_internal method of serializer